src/ralph/back_office/models.py

```diff
@@ -508,6 +510,53 @@ def loan_report(cls, instances, request, **kwargs):
             language=kwargs['report_language']
         )
 
+    @classmethod
+    @transition_action(
+        verbose_name=_('Set as DataCenter Asset'),
+        disable_save_object=True,
+        form_fields={
+            'rack': {
+                'field': forms.CharField(label=_('Rack')),
+                'autocomplete_field': 'rack',
+                'autocomplete_model': 'data_center.DataCenterAsset'
+            },
+            'position': {
+                'field': forms.IntegerField(label=_('Position')),
+            },
+            'model': {
+                'field': forms.CharField(label=_('Model')),
+                'autocomplete_field': 'model',
+                'autocomplete_model': 'data_center.DataCenterAsset'
+            },
+            'service_env': {
+                'field': forms.CharField(label=_('Service env')),
+                'autocomplete_field': 'service_env',
+                'autocomplete_model': 'data_center.DataCenterAsset'
+            }
+        }
+    )
+    def move_as_data_center_asset(cls, instances, request, **kwargs):
+        from ralph.data_center.models.physical import DataCenterAsset, Rack  # noqa
+        with transaction.atomic():
+            for i, instance in enumerate(instances):
+                data_center_asset = DataCenterAsset()
+                try:
+                    move_parents_models(instance, data_center_asset)
+                except Exception as e:
```

Comment:
so IMO we shouldn't catch exception here - instead of it we should catch every exception in transition action call https://github.com/allegro/ralph/blob/ng/src/ralph/lib/transitions/models.py#L270